package.json

```diff
@@ -42,6 +42,16 @@
 				"language": "yaml"
 			}
 		],
+		"configuration": {
+			"properties": {
+				"redhat.telemetry.enabled": {
+					"type": "boolean",
+					"default": null,
+					"markdownDescription": "Enable usage data and errors to be sent to Red Hat servers. Read our [privacy statement](https://developers.redhat.com/article/tool-data-collection).",
+					"scope": "window"
```

Comment:
need to add : `"tags":["telemetry"]`

Reply:
Where?
I have no `tags` completion inside the properties.
![Screenshot from 2022-02-07 14-53-12](https://user-images.githubusercontent.com/1105127/152801064-a732f622-6c0e-4b73-9242-b11567bc29b9.png)

Reply:
Got the info from https://github.com/microsoft/vscode/issues/140972#issuecomment-1016650114. 
It might require a more recent vscode engine dependency for completion to work, or it might be a bug.

Reply:
https://issues.redhat.com/browse/FUSETOOLS2-1490